packages/truffle-db/src/loaders/artifacts/test/index.ts

```diff
@@ -9,26 +9,42 @@ import * as Contracts from "truffle-workflow-compile";
 // mocking the truffle-workflow-compile to avoid jest timing issues
 // and also to keep from adding more time to Travis testing
 jest.mock("truffle-workflow-compile", () => ({
- compile: function(config, callback) {
-   const magicSquare= require(path.join(__dirname, "sources", "MagicSquare.json"));
-   const migrations = require(path.join(__dirname, "sources", "Migrations.json"));
-   const squareLib = require(path.join(__dirname, "sources", "SquareLib.json"));
-   callback(null, {
-     "contracts": [{
-       "contract_name": "MagicSquare",
-       ...magicSquare
-     },
-     {
-       "contract_name": "Migrations",
-       ...migrations
-     },
-     {
-       "contract_name": "SquareLib",
-       ...squareLib
-     }
-     ]
-   });
- }
+  compile: function(config, callback) {
+    const magicSquare= require(path.join(__dirname, "sources", "MagicSquare.json"));
+    const migrations = require(path.join(__dirname, "sources", "Migrations.json"));
+    const squareLib = require(path.join(__dirname, "sources", "SquareLib.json"));
+    const vyperStorage = require(path.join(__dirname, "sources", "VyperStorage.json"));
+    const returnValue = {
+      "outputs": {
+        "solc": [
+          "/Users/fainashalts/solidity-magic-square/contracts/MagicSquare.sol",
+          "/Users/fainashalts/solidity-magic-square/contracts/Migrations.sol",
+          "/Users/fainashalts/solidity-magic-square/contracts/SquareLib.sol"
+        ],
+        "vyper": [
+           "/Users/fainashalts/truffle-six/testing2/contracts/VyperStorage.vy",
+        ]
+      },
+      "contracts": [{
```

Comment:
~Something's off here. As it currently stands, this part of the return value should be an object mapping of contract name -> contract object, not an array. I'm betting the code that's using this mock is wrong as a result... I'll confirm in another pass.~

On another note about these mocks: I know `contract_name` is prevalent, but is that really how we're getting it? The correct field here is `contractName`, although it's possible that it's not coming out of t-w-c like that. I think we should do a `require("truffle-contract-schema").normalize(contractObject)` for all of these.

Reply:
Oh, well, hey! Nevermind! The only place the code looks at the result is [here](https://github.com/trufflesuite/truffle/pull/2036/files#diff-6d89eb0a6dc68a0c021900eb15ee4345R281), and it uses `Object.values(contracts)`, which works for both arrays and objects just fine.

So since this is going to change to using an array anyway, I think it's fine to keep the mock as an array. Don't mind me!